FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/transport/src/plaintext_freertos.c

```diff
@@ -84,13 +87,19 @@ BaseType_t Transport_FreeRTOS_Connect( NetworkContext_t * pNetworkContext,
         if( serverAddress.sin_addr == 0 )
         {
             LogError( ( "Failed to resolve %s.", pHostName ) );
-            socketStatus = -1;
+            socketStatus = TRANSPORT_FREERTOS_NETWORK_ERROR;
         }
     }
 
     if( socketStatus == 0 )
     {
+        LogDebug( ( "Creating TCP Connection to %s.", pHostName ) );
         socketStatus = FreeRTOS_connect( tcpSocket, &serverAddress, sizeof( serverAddress ) );
+
+        if( socketStatus != 0 )
+        {
+            LogError( ( "Failed to establish TCP Connection." ) );
```

Comment:
Log the socket status as well?
```suggestion
            LogError( ( "Failed to establish TCP Connection: ReturnCode=%d", socketStatus ) );
```